openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,680 @@
+import { getCountBadge, getEntityDeleteMessage } from '../../utils/CommonUtils';
+import { getEntityName } from '../../utils/EntityUtils';
+import { DEFAULT_ENTITY_PERMISSION } from '../../utils/PermissionsUtils';
+import {
+  getClassificationDetailsPath,
+  getClassificationTagPath,
+} from '../../utils/RouterUtils';
+import {
+  escapeESReservedCharacters,
+  getEncodedFqn,
+} from '../../utils/StringsUtils';
+import { getQueryFilterToExcludeTerms } from '../../utils/TagsUtils';
+import { showErrorToast, showSuccessToast } from '../../utils/ToastUtils';
+import './tag-page.less';
+import { TagTabs } from './TagPage.inteface';
+
+const TagPage = () => {
+  const { t } = useTranslation();
+  const { fqn: tagFqn } = useFqn();
+  const history = useHistory();
+  const { tab: activeTab = TagTabs.OVERVIEW } = useParams<{ tab?: string }>();
+  const { getEntityPermission } = usePermissionProvider();
+  const [isLoading, setIsLoading] = useState(false);
+  const [tagItem, setTagItem] = useState<Tag>();
+  const [assetModalVisible, setAssetModalVisible] = useState(false);
+  const [isDescriptionEditable, setIsDescriptionEditable] =
+    useState<boolean>(false);
+  const [isNameEditing, setIsNameEditing] = useState<boolean>(false);
+  const [isStyleEditing, setIsStyleEditing] = useState(false);
+  const [isDelete, setIsDelete] = useState<boolean>(false);
+  const [showActions, setShowActions] = useState(false);
+  const [assetCount, setAssetCount] = useState<number>(0);
+  const [tagPermissions, setTagPermissions] = useState<OperationPermission>(
+    DEFAULT_ENTITY_PERMISSION
+  );
+  const assetTabRef = useRef<AssetsTabRef>(null);
+  const [previewAsset, setPreviewAsset] =
+    useState<EntityDetailsObjectInterface>();
+  const breadcrumb: TitleBreadcrumbProps['titleLinks'] = useMemo(() => {
+    return tagItem
+      ? [
+          {
+            name: 'Classifications',
+            url: ROUTES.TAGS,
+            activeTitle: false,
+          },
+          {
+            name: tagItem.classification?.name ?? '',
+            url: tagItem.classification?.fullyQualifiedName
+              ? getClassificationDetailsPath(
+                  tagItem.classification.fullyQualifiedName
+                )
+              : '',
+            activeTitle: false,
+          },
+        ]
+      : [];
+  }, [tagItem]);
+
+  const handleAssetClick = useCallback((asset) => {
+    setPreviewAsset(asset);
+  }, []);
+
+  const { editTagsPermission, editDescriptionPermission } = useMemo(() => {
+    if (tagItem) {
+      return {
+        editTagsPermission:
+          !tagItem.disabled &&
+          (tagPermissions.EditTags || tagPermissions.EditAll) &&
+          !tagItem.deleted,
+        editDescriptionPermission:
+          !tagItem.disabled &&
+          (tagPermissions.EditDescription ||
+            tagPermissions.EditAll ||
+            tagPermissions.EditTags) &&
+          !tagItem.deleted,
+      };
```

Comment:
disabled and deleted  check seems to be repetitive

Reply:
Updated that so the repetition is gone.